src/components/Review/PostCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { css, Theme } from "@emotion/react";
+import { Link } from "@nextui-org/react";
+import { PostMarkdown } from "util/getMarkdownBySlug";
+
+export interface PostListItemProps extends PostMarkdown {}
+
+function PostCard(props) {
+  const post = props;
```

Comment:
이렇게 바인딩하는 것보다 아래처럼 가져갈 수도 있을 거 같아요!!!

```suggestion
function PostCard(post) {
```